### DISCUSSION TITLE
What's the difference between supabase (dart) and supabase_flutter?

### DISCUSSION THREAD

**Discussion by rphly** [supabase/supabase#2426] [Questions] [answered]
Saw that a new Supabase flutter package was created 2 weeks ago, wondering what the difference would be as we're currently using the dart package. Will features diverge? :)

**Comment by dshukertjr** [comment, marked as answer]
Hi @rphly !

`supabase_dart` is a client library that is build with pure Dart.
`supabase_flutter` is a wrapper around `supabase-dart` to solve some of the auth issues for Flutter apps. 
- https://github.com/supabase/gotrue-dart/issues/8
- https://github.com/supabase/gotrue-dart/issues/18

`supabase_flutter` is just there to provide better developer experience for Flutter devs, so the features should differ significantly!

**Reply by rphly** [reply]
Great! Been using supabase_dart for about a month or so now so we've been building around it. When do you think would be a good time to hop over to supabase_flutter?

**Reply by sandbox-apps** [reply]
@rphly what are you building using supabase on flutter? Any blockers using it? Cause i see supabase for flutter is still in early stages?

**Reply by dshukertjr** [reply]
@rphly @sandbox-apps 

Personally, I am thinking about migrating my Flutter apps that I built using `supabase_dart` to `supabase_flutter` soon, but I guess at the end it really depends on personal preferences. Right now, since `supabase_flutter` is in its early stage, there is a lot of room for improvements, which means lot more opportunities to give feedback and help grow the project! Personally, I like doing that kind of things, so don't mind it being in early stages, but I do understand that some people prefer solid products that have been around for a while.

**Reply by jtkeyva** [reply]
Any progress on Supabase Dart handling authentication or Supabase Flutter supporting all platforms?

**Reply by dshukertjr** [reply]
@jtkeyva There are some exciting things happening in supabase-flutter! Once we merge [this](https://github.com/supabase-community/supabase-flutter/pull/124) PR that brings a better developer experience, we will work on a fix for [this](https://github.com/supabase-community/supabase-flutter/issues/106) issue, which will add desktop app support to supabase-flutter!

**Reply by jtkeyva** [reply]
great to hear! you guys are doing a great job keeping up with the demand
